ui/views/components/footer.ejs

```diff
@@ -0,0 +1,34 @@
+<footer class="footer p-6">
+  <div class="columns is-vcentered is-mobile">
+    <div class="column">
+      <a href="/">
+        <img src="../../assets/images/logo_footer.png" alt="Ardana" />
+      </a>
+    </div>
+
+    <div class="column">
+      <div class="mt-6 is-pulled-right">
+        <div class="columns is-mobile">
+          <div class="column">
+            <img src="../../assets/images/twitter.png" />
```

Comment:
PNGs??

Reply:
will replace them with SVGs

Reply:
turns out the SVGs are larger than PNGs

```
[nix-shell:~/Sandbox/Danaswap/ui]$ ls -alh !$
ls -alh assets/images/twitter.png
-rw-r--r-- 1 hao users 624 Oct 28 09:53 assets/images/twitter.png

[nix-shell:~/Sandbox/Danaswap/ui]$ ls -alh assets/images/twitter.svg 
-rw-r--r-- 1 hao users 1.1K Oct 28 09:43 assets/images/twitter.svg
```

Reply:
I couldn't find this SVG in the project. But [this Twitter icon](https://iconmonstr.com/twitter-1-svg/) I found online is 607 bytes.

You could also try running it through `scour` for production and seeing how much smaller it is.

```console
$ scour input.svg output.svg --enable-comment-stripping --enable-viewboxing --indent=none
```

Regardless here, we would need to ship a `2x` version because it *will* look blurry on hiDPI screens like my laptop or most smartphones. The 2x will almost certainly be bigger than the SVGs in these cases. Using just the SVG works for almost every use case which would reduce the overall size. I would work the other way: I would go SVGs and then circle back to raster graphics as an optimization if it possibly shaves a few bytes on some displays. I could see a situation where you want for smaller images. Here's one for loDPI displays sending a AVIF file:

```html
<picture>
  <source srcset="file.avif" type="image/avif" media="(max-device-pixel-ratio: 1.2)">
  <img src="file.svg">
</picture>
```

Reply:
@toastal the assets are provided here https://www.figma.com/file/5KTUxesjnRIV8YM62dSosc/DanaSwap?node-id=4%3A134